packages/shared/src/common/date.ts

```diff
@@ -11,8 +12,30 @@ export function isValidISODate(date: string): boolean {
   return buildDayjs(date, ISO_DATE, true).isValid();
 }
 
-const isValidISODateOptional = (date: string | undefined | null): boolean =>
-  date ? isValidISODate(date) : true;
+function isValidISODateOptional(date: string | undefined | null): boolean {
+  return date ? isValidISODate(date) : true;
+}
+
+export function validateIsPast(date: string): boolean {
+  if (dayjs(date).isAfter(dayjs())) {
```

Comment:
re-use `validateIsPastSafe`?

Reply:
Yeah, good call. Will do in another PR/release